docs/source/whatsnew.rst

```diff
@@ -1,338 +1,56 @@
-What's new in Hyperledger Fabric v2.x
-=====================================
+What's new in Hyperledger Fabric
+================================
 
-What's New in Hyperledger Fabric v2.5
+What's New in Hyperledger Fabric v3.0
 -------------------------------------
 
-Purge history of private data
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Byzantine Fault Tolerant (BFT) ordering service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-While it has always been possible to delete private data from current state, this new feature enables purging the history of private data from a peer while preserving a hash of the private data as immutable evidence on the blockchain.
+Hyperledger Fabric has utilized a Raft crash fault tolerant (CFT) ordering service since version v1.4.
+A Byzantine Fault Tolerant (BFT) ordering service can withstand not only crash failures, but also a subset of nodes behaving maliciously.
+Fabric v3.0 is the first release to provide a BFT ordering service based on the
+`SmartBFT <https://arxiv.org/abs/2107.06922>`_ `consensus library <https://github.com/SmartBFT-Go/consensus>`_.
+Consider using the BFT orderer if true decentralization is required,
+where up to and not including a third of the parties running the orderers may not be trusted due to malicious intent or being compromised.
 
-* Useful for purging private data on demand for privacy reasons or to adhere to government regulations.
-* Deletes private data from state and from peer’s private data history so that it can no longer be queried from block events or from other peers.
-* Available as a new chaincode API `PurgePrivateData()`.
-* Requires setting application capability to `V2_5` in channel configuration
+Learn more about the BFT ordering service in the **BFT** section of the :doc:`orderer/ordering_service` documentation.
 
-For more details, see the :doc:`private-data/private-data` topic.
+For more details see the :doc:`bft_configuration` topic.
 
-Multi-architecture binaries and docker images are now available
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Try it out for yourself! Look for the **Bring up the network using BFT ordering service** in the :doc:`test_network` tutorial.
```

Comment:
maybe reference the exact location?

https://hyperledger-fabric--4418.org.readthedocs.build/en/4418/test_network.html#bring-up-the-network-using-bft-ordering-service

Reply:
I don't think you can reference specific sections in RST relative links... if somebody knows how do tell!